draft-ietf-moq-transport.md

```diff
@@ -1307,59 +1344,62 @@ GOAWAY Message {
   If a server receives a GOAWAY with a non-zero New Session URI Length it MUST
   terminate the session with a Protocol Violation.
 
-## MAX_SUBSCRIBE_ID {#message-max-subscribe-id}
+## MAX_SUBSCRIPTION_ID {#message-max-subscribe-id}
 
-A publisher sends a MAX_SUBSCRIBE_ID message to increase the number of
+A publisher sends a MAX_SUBSCRIPTION_ID message to increase the number of
 subscriptions a subscriber can create within a session.
 
-The Maximum Subscribe Id MUST only increase within a session, and
-receipt of a MAX_SUBSCRIBE_ID message with an equal or smaller Subscribe ID
-value is a 'Protocol Violation'.
+The Maximum Subscriber-Chosen Subscription ID MUST only increase within a
+session, and receipt of a MAX_SUBSCRIPTION_ID message with an equal or smaller
+Subscription ID value is a 'Protocol Violation'.
 
 ~~~
-MAX_SUBSCRIBE_ID
+MAX_SUBSCRIPTION_ID
 {
   Type (i) = 0x15,
   Length (i),
-  Subscribe ID (i),
+  Subscription ID (i),
 }
 ~~~
-{: #moq-transport-max-subscribe-id format title="MOQT MAX_SUBSCRIBE_ID Message"}
+{: #moq-transport-max-subscribe-id format title="MOQT MAX_SUBSCRIPTION_ID Message"}
 
-* Subscribe ID: The new Maximum Subscribe ID for the session. If a Subscribe ID
-{{message-subscribe-req}} equal or larger than this is received by the publisher
-that sent the MAX_SUBSCRIBE_ID, the publisher MUST close the session with an
-error of 'Too Many Subscribes'.
+* Subscription ID: The new Maximum Subscriber-Chosen Subscription ID for the
```

Comment:
So this limits subscribers, but not publishers?  Do we want to use even/odd here to differentiate the limits like we do in QUIC?

Reply:
You can't do that because PSID is not restricted to increase - they can be sprayed all over the 61 bit space.  We could limit control messages using something like control message IDs (moq-transport PR 898), but I'm not sure how we can limit publisher initiated subscriptions.